daemon/config/config.go

```diff
@@ -22,6 +22,9 @@ type Config struct {
 	// Network config
 	NetworkConfg network.Config
 
+	// Whether enable cri manager.
+	CriEnabled bool
```

Comment:
Please add a json name for this field, since it will also be included in the config file which in json, like ``json:"home-dir,omitempty"``.